src/vocab.rs

```diff
@@ -75,4 +75,16 @@ impl PySequenceProtocol for PyVocab {
             Ok(words[idx as usize].clone())
         }
     }
+
+    fn __contains__(&self, word: String) -> PyResult<bool> {
+        let embeds = self.embeddings.borrow();
+        Ok(embeds
+            .vocab()
+            .idx(&word)
+            .map(|word_idx| match word_idx {
+                WordIndex::Word(_) => true,
+                WordIndex::Subword(_) => false,
+            })
+            .unwrap_or(false))
```

Comment:
`is_some()`

Reply:
Good catch

Reply:
Actually, `is_some()` conveys that the word is in-vocab if there's a `WordIndex::Subword` returned. We'd only return `false` if no subword indices can be generated for a word. I'd like to return false for any word that isn't part of the actual vocabulary

Reply:
Ah right. But this can be made nice with `is_some()` with the new `WordIndices` changes in `finalfusion-rust`.

Reply:
Only with a bumped dependency ;)